### ISSUE TITLE
Missing files when installing with bower or npm

### ISSUE THREAD

**Issue by jperals** [qmlweb/qmlweb#425]
After cloning and building qmlweb via `npm run build`, the content of the `lib` directory is:

```
LICENSE
qmlweb.js
qmlweb.js.map
qmlweb.min.js
qmlweb.min.js.map
qmlweb.parser.js
qmlweb.parser.js.map
qmlweb.parser.min.js
qmlweb.parser.min.js.map
qt.js
qt.js.map
qt.min.js
qt.min.js.map
```

But when installing as a dependency with bower or npm, the content of the `lib` directory is:

```
LICENSE
qmlweb.parser.js
qmlweb.parser.js.map
qmlweb.parser.min.js
qmlweb.parser.min.js.map
qt.js
qt.js.map
qt.min.js
qt.min.js.map
```

So these files are missing:
```
qmlweb.js
qmlweb.js.map
qmlweb.min.js
qmlweb.min.js.map
```

Which correspond to the actual file that needs to be included from the HTML markup, so it is actually impossible (or am I missing something?) to use the library from the bower or npm packages.

Using npm 5.4.2 and bower 1.8.0 on Mac OS High Sierra.

[This bug](https://github.com/qmlweb/qmlweb/issues/423) might be related, if not the same.

**Comment by jperals** [comment]
It looks to me that the packages are just not up to date.

**Comment by ChALkeR** [comment]
Yes, in master those were renamed, and in the last released version, v0.2.0, there are no such files yet.

**Comment by ChALkeR** [comment]
Those instruction is for the `master` branch. For the last published version (v0.2.0) the corresponding file were historically named `qt.js` and `qt.min.js`. See https://github.com/qmlweb/qmlweb/tree/v0.2.0
Also the readme [at npm](https://www.npmjs.com/package/qmlweb) corresponds to the version published there.

That will be changed to `qmlweb.js` once v0.3.0 will be published.

I'm closing this issue as I think that answers the question, feel free to ping me if you think that this should be reopened. :wink:
